frontend/src/metabase/visualizations/visualizations/Map.jsx

```diff
@@ -341,3 +349,27 @@ export default class Map extends Component {
     }
   }
 }
+
+const mapStateToProps = (state, props) => {
+  return {
+    isAdmin: getUserIsAdmin(state, props),
+  };
+};
+
+const CustomMapFooter = connect(mapStateToProps)(({ isAdmin }) => (
+  <CustomMapContainer>
+    <Button
```

Comment:
I didn't get it why we used `Button` here and `Link` for isAdmin=true and `ExternalLink` otherwise

Reply:
I attempted to wrap it in a link, but I got errors around the link element being used outside of a `<Router />`. Not totally sure why though

Reply:
If that's the case then `Button` isn't a solution either. We need to investigate this further

Reply:
That's fair. I'll dig a little deeper and see what I come up with

Reply:
I think we should try to make `Link` work. Not sure why it didn't work either.

Reply:
Working on it now. I was trying to use the link from `react-router-dom`, not our core components. but things are working now. PR will be updated in 10 min